tests/installation/grub_test.pm

```diff
@@ -59,7 +59,7 @@ sub run {
     }
     workaround_type_encrypted_passphrase;
     # 60 due to rare slowness e.g. multipath poo#11908
-    assert_screen "grub2", 60;
+    assert_screen "grub2", 300;
```

Comment:
Are you sure this also happened in production? It will happen when you do tests locally with loading from qemu screenshots so that you never finish the normal installation from start to end. Then your machine boots from DVD as well then.